src/xdocs/google_style.xml

```diff
@@ -665,7 +665,7 @@
                             <td><a name="4.6.1"/><a href="#4.6.1"><img src="images/anchor.png" alt=""/></a></td>
                             <td>
                                 <a
-                                    href="http://checkstyle.sourceforge.net/reports/google-java-style.html#s4.6.1-vertical-whitespace">4.6.1 Vertical Whitespace</a>
+                                    href="http://checkstyle.sourceforge.net/reports/google-java-style-20160712.html#s4.6.1-vertical-whitespace">4.6.1 Vertical Whitespace</a>
```

Comment:
from guide:
> Exception: A blank line between two consecutive fields (having no other code between them) is optional. Such blank lines are used as needed to create logical groupings of fields.

We can not satisfy this exception. we need to update coverage report to show image as "Existing Check covers some part of requirements from Google." as we can not completely cover it.